src/main/java/FlightList.groovy

```diff
@@ -0,0 +1,57 @@
+import java.time.LocalDateTime
+
+interface FlightList {
+    List<Flight> getAllFlightsTo(Airports destination)
+
+    List<Flight> getAllFlightsFrom(Airports from)
+
+    void printFlights()
+
+    void printFlightsTo(Airports destination)
+
+    void printFlightsFrom(Airports to)
```

Comment:
This should be a simple "println listOfFlights" you can get a list of flights the above methods that return a List<Flights>
I don't see much value in these methods